app/scripts/TrackRenderer.js

```diff
@@ -381,6 +381,17 @@ class TrackRenderer extends React.Component {
   }
 
   componentDidUpdate(prevProps) {
+    // If the initial domain changed, a new view config
+    // probably has loaded. Reset the element's zoomTransform in this case.
+    if (
+      this.props.initialXDomain[0] !== prevProps.initialXDomain[0] ||
+      this.props.initialXDomain[1] !== prevProps.initialXDomain[1] ||
+      this.props.initialYDomain[0] !== prevProps.initialYDomain[0] ||
+      this.props.initialYDomain[1] !== prevProps.initialYDomain[1]
+    ) {
+      this.element.__zoom = zoomIdentity;
```

Comment:
A short comment what `__zoom` is used for would be useful in case we look at the code in the future :)